docs/DocumentationStyleGuide/accessibility.rst

```diff
@@ -0,0 +1,94 @@
+
+The general guidelines for creating accessible documentation pieces are:
+
+-  Maintain consistency across the documentation. Both in style and
+   visuals by employing a fixed hierarchy.
+-  Avoid language or word usage that reeks of bias or a disregard for a
+   group of people.
+-  Focus on the audience that you are writing for and follow a
+   conversation style of documentation.
+-  Define acronyms, abbreviations, prerequisites initially and never
+   assume the technical competency of the reader.
+-  Use alternative text for multimedia elements wherever possible, and
+   be conscious of the usage of visual elements.
+
+Image accessibility
+-------------------
+
+-  Validate that the images, diagrams, graphs, icons possess a
+   meaningful alternative-text description for a meaningful equivalent.
+-  Use alternative text to describe what the image, digraph, graph or
+   icon does, rather than what it looks like.
+-  All images, diagrams, graphs, icons should contain a short image
+   description that is self-explanatory.
+-  Prefer the usage of SVGs over JPEGs and PNGs. SVGs scale on zooming
+   and provide better context to the readers.
+-  Don’t use images of text, terminal outputs, code samples and other
+   information that can be displayed as text.
+
+Text accessibility
+------------------
+
+-  Follow a content hierarchy defined by the content strategy or the
+   documentation plan for the project.
+-  Use Heading-1 only once in the entire content piece. Use Heading-2
+   and Heading-3 for all the sub-headings (#, ##, ### in Markdown).
+-  Headings should be correctly nested to indicate their relative
+   significance in the documentation piece.
+-  Avoid Camel-Case, inconsistent formatting and unnecessary
+   capitalization of text.
+-  Always break down your documentation piece into short paragraphs that
+   are easy to consume in chunks and pieces.
+
+UI accessibility
+----------------
+
+-  Avoid documenting instructions and steps that rely on sensory
+   characteristics of the UI and instead rely on what it is supposed to
+   do.
+-  Document the information that is presented in visual colours, in text
+   format as well.
+-  Avoid directional indicators, like above, below, top, while
+   documenting User Interface Navigation.
+-  Provide necessary feedback to designers and engineers on
+   accessibility issues.
+
+Links accessibility
+-------------------
+
+-  Use meaningful link texts with proper Uniform Resource Locator (URL)
+   slugs.
+-  Add an indicator, if a link redirects to an external website and
+   opens on another tab, with a text or an icon.
+-  If a link downloads an external file, mention the file type, its
+   standard purpose and the necessary prerequisites.
+-  Don’t use conventional verbs like *here*, *click on this link*, *read
+   this* which betrays the needful necessary to the user before clicking
+   anything.
+-  Use a tool like `broken-link-checker`_ to check for broken links on
```

Comment:
This may be a different section, but rules about how to format links etc are also useful.

Reply:
That would go in the content strategy. It would be our single resource to refer to how to do the writing.